src/TSHelper.ts

```diff
@@ -659,11 +659,12 @@ export function getFunctionContextType(type: ts.Type, checker: ts.TypeChecker):
     return reduceContextTypes(signatureDeclarations.map(s => getDeclarationContextType(s, checker)));
 }
 
-export function escapeString(text: string): string {
+export function escapeString(text: string, exclude: RegExp[] = []): string {
```

Comment:
I don't really like ability to specify any unrelated regexps there.
Maybe just a `multiline` flag?